src/receiver/index.ts

```diff
@@ -57,10 +58,10 @@ export class ExpressReceiver extends EventEmitter implements Receiver {
       this.emitHandler.bind(this),
     ];
 
-    const e = Object.values(this.endpoints);
+    const endpointPaths = Object.values(this.endpoints);
 
-    for (const endpoint in e) {
-      this.app.post(endpoint, ...defaultMiddleware);
+    for (const endpoint in endpointPaths) {
```

Comment:
`for...of` is usually better for when you want the values not the keys, but nbd.